cypress/e2e/cloud/home.test.ts

```diff
@@ -1,8 +1,12 @@
+import {Organization} from '../../../src/types'
+
+let orgID = ''
```

Comment:
I think the best practice for this kind of thing is to use `cy.wrap`. https://docs.cypress.io/api/commands/wrap#Syntax Otherwise it's possible this variable might not get assigned when you think it should because of how cypress handles its async events with its internal scheduler

Reply:
not sure i follow, since the operation should be set prior to the test running, unless i'm misunderstanding beforeEach. I'll take a look at the docs though since wrapping in a get had caused issues with intercepting requests

Reply:
i also dont understand why you're using this technique and not the `cy.get('@org')` around the intercept

Reply:
https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Variables It looks like it's ok in some cases - so maybe this situation is alright? But typically you're not supposed to set global variables like you're doing here with orgID

Reply:
oh lol you're right @drdelambre - it's already been wrapped. So `cy.get('@org')` should be all that's needed

Reply:
here's an example where using global variables was causing flaky behavior: https://github.com/influxdata/ui/pull/3377

Reply:
Right, I used the get and it messed with intercept is what I'm saying